src/autocomplete/src/Autocomplete.js

```diff
@@ -48,9 +48,22 @@ export default class Autocomplete extends PureComponent {
     selectedItem: PropTypes.any,
 
     /**
-     * The selected item to be selected & shown by default on the autocomplete
+     * The selected item to be selected & shown by default on the autocomplete (deprecated)
+     * @return {Error} a deprecation warning
      */
-    defaultSelectedItem: PropTypes.any,
+    defaultSelectedItem: () =>
+      new Error(
```

Comment:
Should we adopt the same pattern seen elsewhere: https://github.com/thefrontside/deprecated-prop-type/blob/master/deprecated.js#L5

Reply:
Never seen that. 

I’m a little confused as to why you would want to validate a deprecated prop’s type.

Here using `defaultSelectedItem` will always display an error in your console – whatever type you pass to it. Then `initialSelectedItem`’s prop-type from `Downshift` will warn you if your type is invalid.

But if you want to use that, I’ll change. :)

Reply:
I think it's more about only showing the deprecation warning once per message.